### PR TITLE
Sc 17819

### PR DESCRIPTION
**Pull request by daniellemaxwell**
Adds placeholders for the data sources to test the detail and list views. Also adds suggested changes to the 
`data-playground` archetypes following comments from PR #165.

Figma link: https://www.figma.com/file/LNW4E3VcdnEgR7s1QO0gep/Rotational-Website-Refactor?type=design&node-id=1885-215&t=Mpz1wxW3rketzhHH-0

Fixes SC-17819

Acceptance Criteria:
Note: The video below only displays the list view as the single page view is currently in progress.
https://www.awesomescreenshot.com/video/18163731?key=f1083b740a6753bf131ed4d721ea8cb4

### CODE REVIEW

File: archetypes/data-playground.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 ---
 title: "{{ replaceRE "^[-0-9]{10}-" "" .Name | replaceRE "-" " " | title }}"
 slug: "{{ replaceRE "^[-0-9]{10}-" "" .Name }}"
-subtitle: "add subtitle here" 
+subtitle: "Add Data Type Here"
```
Comment (daniellemaxwell) [inline archetypes/data-playground.md:4]: I believe that I added @bbengfort suggestions. I also tried to make the page a little more descriptive after it was mentioned during our last meeting. The order has been amended to match the Figma design to hopefully make things easier as changes are made and for reviewers.

**Review by masskoder — APPROVED** [review]

File: archetypes/data-playground.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+license: "Add Data Source License Type Here (Ex. Free, Commercial)"
+is_api_key_required: false
+api_type: "Add API Type(s) Provided Here and Separate Multiple Types With A Comma"
+sdks: "Add Available SDKs Here and Separate Multiple SDKs With A Comma"
+limits: "Add API Limit Information Here, If Available"
 
 ---
 
-Add data products showcase 
+<!-- Add content for data playground here, including a table for data products if available -->
```
Comment (masskoder) [inline archetypes/data-playground.md:22]: why this line is commented out ?

Comment (daniellemaxwell) [reply archetypes/data-playground.md:22]: To add @bbengfort suggestion from here: https://github.com/rotationalio/rotational.io/pull/165#discussion_r1223346334

Comment (bbengfort) [reply archetypes/data-playground.md:22]: This is guidance for the author; the archetype is a template whose audience is the author of a data source; we can add sections and headings but should also add comments to explain how to create a good data playground single page.

**Pull request merged**